packages/web-react/src/components/UNSTABLE_Toggle/README.md

```diff
@@ -0,0 +1,107 @@
+# UNSTABLE Toggle
+
+> ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
+> Please use it with caution.
+
+Toggle is a form control that allows users to switch between two states.
+
+## Basic Usage
+
+The Toggle component implements the HTML [checkbox input][mdn-checkbox] element. It uses
+the native input element and styles it to look like a toggle switch.
+
+```jsx
+import { UNSTABLE_Toggle } from '@lmc-eu/spirit-web-react/components';
+
+<UNSTABLE_Toggle id="toggle-default" label="Toggle Label" />;
+```
+
+## Indicators
+
+If you need to indicate the state of the toggle, you can add the `hasIndicators` prop. This will add a visual indicators to the toggle switch.
+
+```jsx
+<UNSTABLE_Toggle id="toggle-indicators" label="Toggle Label" hasIndicators />
+```
+
+## Required
+
+Add the `isRequired` prop to mark it as required.
+
+```jsx
+<UNSTABLE_Toggle id="toggle-required" label="Toggle Label" isRequired />
+```
+
+## Hidden Label
+
+```jsx
+<UNSTABLE_Toggle id="toggle-hidden-label" label="Toggle Label" isLabelHidden />
+```
+
+## Fluid
+
+```jsx
+<UNSTABLE_Toggle id="toggle-fluid" label="Toggle Label" isFluid />
+```
+
+## Helper Text
+
+```jsx
+<UNSTABLE_Toggle id="toggle-helper-text" label="Toggle Label" helperText="Helper text" />
+```
+
+## Validation States
+
+Validation states can be presented either by prop `validationState`. See Validation state [dictionary][dictionary-validation].
+
+```jsx
+<UNSTABLE_Toggle id="toggle-success" label="Toggle Label" validationState="success" />
+<UNSTABLE_Toggle
+  id="toggle-warning"
+  label="Toggle Label"
+  validationText="Validation text"
+  validationState="warning"
+  isChecked
+/>
+<UNSTABLE_Toggle
+  id="toggle-danger"
+  label="Toggle Label"
+  validationText={['First validation text', 'Second validation text']}
+validationState="danger"
+/>
+```
+
+## Disabled State
+
+On top of adding the `isDisabled` props to disable Toggle.
+
+```jsx
+<UNSTABLE_Toggle id="toggle-disabled" label="Toggle Label" isDisabled />
+```
+
+## API
```

Comment:
Aren't `autoComplete` and `name` props missing?

Reply:
add name to demo. Otherwise, it is possible to send additional attributes in restprops

Reply:
But it is explicitly mentioned in the documentation of the every form component 👀

Reply:
OK.  I also added mention to the API table in the documentation.